mithril-common/src/messages/message_parts/signer.rs

```diff
@@ -194,6 +207,57 @@ impl SignerMessagePart {
     }
 }
 
+impl TryInto<Signer> for SignerMessagePart {
```

Comment:
Maybe you can also implement the `TryInto<SIgnerWithStake>` trait to align implementations done for `Signer` and `SignerWithStake`?

Reply:
Done